ui/components/app/dropdowns/network-dropdown.js

```diff
@@ -294,27 +309,48 @@ class NetworkDropdown extends Component {
         }}
       >
         <div className="network-dropdown-header">
-          <div className="network-dropdown-title">
-            {this.context.t('networks')}
-          </div>
+          <div className="network-dropdown-title">{t('networks')}</div>
           <div className="network-dropdown-divider" />
-          <div className="network-dropdown-content">
-            {this.context.t('defaultNetwork')}
-          </div>
+          {showTestnetMessageInDropdown ? (
+            <div className="network-dropdown-content">
+              {t('defaultNetwork', [
+                <span key="testNetworksEnabled">
+                  {shouldShowTestNetworks ? t('disable') : t('enable')}
+                </span>,
+                <span
+                  key="advancedSettingsLink"
+                  className="network-dropdown-content--link"
+                  onClick={() => {
```

Comment:
Please do not use spans for buttons! This makes them impossible to focus with a keyboard. Semantic HTML is accessible HTML.

Reply:
this was an oversight, thanks for pointing it out, fix pushed

Reply:
woops, one more thing to push

Reply:
Good catch!  Sorry I didn't see this!

Reply:
updated thanks!